### ISSUE TITLE
Predefining "Distilled CFG Scale" & "Sampling Methods" for ui-config.json file gets ignored

### ISSUE THREAD

**Issue by Lumerica** [lllyasviel/stable-diffusion-webui-forge#1891]
I always put the custom configurations I need in the ui-config.json configuration file.

This saves me time and I would not have to change every settings every time.
Many configurations works usign this way, however for some reason it does not work for sampling method and distilled CFG scale.

For example. Dispites that I've set the below value in the file, it will just keep reusing the default value somewhere which is 3.5, same goes for sampling method, I've set it to DEIS, but it always pick Euler:
     "txt2img/Distilled CFG Scale/value": 4.5,
    "customscript/sampler.py/txt2img/Sampling method/value": "DEIS",

Additionally, it would be nice if we can just set a static value for the GPU weight either via configuration (config.json) or ui-config.json so that it does not keep automatically set the the GPU weight to maximum whenever you switch the UI layout (via the top left corner).

Btw, With this nice option to switch UI layout around, it does however meddle with the settings defined and switching it around will either reset some settings (like the "GPU weight", and "Diffusion in Low Bits"). So if we can set this all in the configuration file it would be nice.
Preferably that we can actualyl define for each layout a different predefined step, sampling method, CFG configuration, and any settings depending on which UI layout is configured).

**Comment by grandfromages** [comment]
Yes, the Default settings for Sampler and Schedule Type, specifically don't get saved.

**Comment by Lumerica** [comment]
> Yes, the Default settings for Sampler and Schedule Type, specifically don't get saved.

The suggestion is:
if it can have a specific parameter separated per UI option (SDXL/SD/Flux/All) in the file we can predefine it in the ui-config file
